verticapy/tests/vDataFrame/test_vDF_statistical_tests.py

```diff
@@ -1,33 +0,0 @@
-# (c) Copyright [2018-2020] Micro Focus or one of its affiliates.
```

Comment:
Why have you deleted the file?

Reply:
I changed it to test_vDF_tsa_tools as the stat tests were moved from the vDataFrame.